### ISSUE TITLE
Can not download file whose filename is in chinese

### ISSUE THREAD

**Issue by bit-jmm** [Requarks/wiki#352]
### Actual behavior

Can not download file whose filename is in chinese

### Steps to reproduce the behavior

upload a file whose filename is in chinese，then download it，not found occur。

**Comment by NGPixel** [comment]
Non-latin characters are not supported in search and filenames in 1.x.

Full Chinese support will come in 2.0.
